.github/workflows/release-changeset-changelog.yml

```diff
@@ -0,0 +1,37 @@
+name: Generate changelog on v2-release
+on:
+  push:
+    branches:
+      - v2-release/**
+
+jobs:
+  release-canary:
+    name: Release canary package
+    needs: [build, build-release, cypress-react, cypress-vue]
+    if: ${{ github.ref == 'refs/heads/v2-develop' }}
+    # Can't use 'configure-enviroment' because it fails on dev branch
```

Comment:
Do you know why it fails?

Reply:
```
          else
            exit 1
          fi
```

Reply:
but it doesnt matter, I pushed old version, now this whole file looks different